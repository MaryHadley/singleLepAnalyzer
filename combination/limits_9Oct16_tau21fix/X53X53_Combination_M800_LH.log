000 (1295362.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295362.000.000) 10/09 22:41:05 Job executing on host: <192.168.63.235:42190>
...
006 (1295362.000.000) 10/09 22:41:14 Image size of job updated: 188484
	86  -  MemoryUsage of job (MB)
	87284  -  ResidentSetSize of job (KB)
...
006 (1295362.000.000) 10/09 22:46:15 Image size of job updated: 283464
	141  -  MemoryUsage of job (MB)
	144040  -  ResidentSetSize of job (KB)
...
006 (1295362.000.000) 10/10 00:01:22 Image size of job updated: 283464
	141  -  MemoryUsage of job (MB)
	144124  -  ResidentSetSize of job (KB)
...
006 (1295362.000.000) 10/10 18:33:28 Image size of job updated: 297752
	156  -  MemoryUsage of job (MB)
	158748  -  ResidentSetSize of job (KB)
...
005 (1295362.000.000) 10/10 18:38:38 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 19:55:56, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 19:55:56, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1590  -  Run Bytes Sent By Job
	306  -  Run Bytes Received By Job
	1590  -  Total Bytes Sent By Job
	306  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31866        1   3824811
	   Memory (MB)          :      156     3072      3072
...
