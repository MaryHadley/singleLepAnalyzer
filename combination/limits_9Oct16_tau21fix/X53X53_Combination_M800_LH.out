/var/lib/condor/execute/dir_1793
Total number of likelihood evaluations: 1100000099
Total number of likelihood evaluations: 11000099
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M800_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M800_LH/quant-toys:0-sig-e857f98791.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M800_LH/quant-data-sig-594b8db276.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M800_LH
