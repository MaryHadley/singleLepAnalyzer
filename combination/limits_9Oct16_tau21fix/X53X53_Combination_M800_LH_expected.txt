# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0     0.1105   0.055303    0.23054  0.076058    0.16227
